src/Translators/Rules.php

```diff
@@ -94,4 +106,33 @@ private static function overrideStringRuleForSpecialNames($name)
 
         return 'string';
     }
+
+
+    private static function betweenRuleForNumericTypes(Column $column, $numericType)
+    {
+        $defaultPrecisionAndScale = [8, 2];
```

Comment:
If there is no _precision_ or _scale_, I wouldn't want to add the `between` rule.